lib/runner/extensions/request-wrapper.command.js

```diff
@@ -0,0 +1,32 @@
+var _ = require('lodash');
+
+module.exports = {
+    init: function (done) {
+        done();
+    },
+
+    triggers: ['response'],
+
+    process: {
+        requestWrapper: function (payload, next) {
+            var abortOnError = _.has(payload, 'abortOnError') ? payload.abortOnError : this.options.abortOnError,
+                complete = function (nextPayload, err) {
+                    this.triggers.response(
```

Comment:
resolve variables here

Reply:
Not trivial to include in this PR. Will do it separately.